src/OpenTelemetry.Exporter.Geneva/GenevaLogExporter.cs

```diff
@@ -44,6 +45,8 @@ public class GenevaLogExporter : GenevaBaseExporter<LogRecord>
     };
 
     private readonly IDataTransport m_dataTransport;
+    private static readonly ConcurrentDictionary<string, string> rawNameToSanitizedName = new();
```

Comment:
General tip: if the capacity is known ahead of time, use the ctor which takes capacity.

e.g. https://docs.microsoft.com/en-us/dotnet/api/system.collections.concurrent.concurrentdictionary-2.-ctor?view=net-6.0#system-collections-concurrent-concurrentdictionary-2-ctor(system-int32-system-int32)

Reply:
Use a more descriptive name such as `implicitTableNameMapping`.

Reply:
https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/345/files/8ab761d635112f7640fcf589e5f873cedf4c2724#r875366580